### PR TITLE
chore(table): replace lodash-es with lodash

### LINKED ISSUE #2616: [PF4] lodash-es problems in React-table
Hi

After add the react-table for the VirtualScroll in kiali we have a problem with lodash-es library imported in react-table.

Why react-table use lodash-es instead lodash like react-charts?
```bash
  This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /home/aljesusg/Projects/kiali/kiali-ui/node_modules/@patternfly/react-table/node_modules/lodash-es/lodash.js:10
    export { default as add } from './add.js';
    ^^^^^^

    SyntaxError: Unexpected token export

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:471:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:513:25)
      at Object.<anonymous> (node_modules/@patternfly/react-table/dist/js/components/Table/base/evaluate-transforms.js:8:17)
```

Thanks

### LINKED ISSUE #2617: Replace lodash-es with lodash
We have no reason to depend twice on the same library. We should replace *`lodash-es` with `lodash`.

### PR DESCRIPTION
**Pull request by redallen**
**What**: Fixes #2617 Fixes #2616

**Additional issues**:

### CODE REVIEW

File: packages/patternfly-4/react-virtualized-extension/package.json [hunk 1]
```diff
@@ -40,7 +40,6 @@
   },
   "peerDependencies": {
     "@patternfly/react-table": "^2.10.0",
-    "lodash-es": "4.x",
```
Comment (priley86) [inline packages/patternfly-4/react-virtualized-extension/package.json]: should `lodash` be added as a `peerDependency` now?

Comment (redallen) [reply packages/patternfly-4/react-virtualized-extension/package.json]: `react-virtualized-extension` doesn't use `lodash-es` anywhere.

**Comment by priley86** [comment]
this is nearly identical to #2640  now... let's pick one 😄

**Comment by aljesusg** [comment]
This Fix #2616 too.

**Comment by aljesusg** [comment]
Hooray !!! Thanks

**Pull request merged**